portability-core/src/main/java/org/dataportabilityproject/serviceProviders/microsoft/MicrosoftServiceProvider.java

```diff
@@ -61,14 +59,21 @@
         .put(ServiceMode.EXPORT, ImmutableList.of(PortableDataType.CALENDAR, PortableDataType.MAIL))
         .build();
 
+    private final static Map<PortableDataType, ListMultimap<ServiceMode, String>> DATA_TYPE_SCOPES =
+        ImmutableMap.<PortableDataType, ListMultimap<ServiceMode, String>>builder()
+            .put(PortableDataType.CALENDAR, ImmutableListMultimap.<ServiceMode, String>builder()
+                .putAll(ServiceMode.EXPORT, Arrays.asList("wl.calendars", "wl.contacts_calendars"))
```

Comment:
Not sure if it's any cleaner but with multipmaps you can continue to add values to the same key. Whatever you think is cleaner to read, even if you don't want to use multimaps, just a suggestion.
.put(ServiceMode.EXPORT, "wl.calendars")
.put(ServiceMode.EXPORT,  "wl.contacts_calendars")

Reply:
Oh I see - I was thinking it was cleaner and less error prone to put them all on one line. So if the scopes change, the place to modify is clearer.